src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
```

Comment:
Spacing shouldn't be changed in this file

Reply:
It's fixed.